specutils/spectra/spectrum_collection.py

```diff
@@ -9,6 +9,9 @@
 
 __all__ = ['SpectrumArray', 'SpectrumCollection']
 
+__doctest_skip__ = ['specutils.spectra.spectrum_collection.ResampleMixin', 
+                    'astropy.nddata.mixins.ndio.NDIOMixin']
```

Comment:
why do you need to list the astropy class here?

Reply:
I'm not sure. The failing doc test was raising it as a warning (`/home/travis/build/astropy/specutils/docs/api/specutils.SpectrumCollection.rst:6: WARNING: py:class reference target not found: astropy.nddata.mixins.ndio.NDIOMixin`). It's still doing it, so this didn't really do anything (I'll be reverting that commit).

Reply:
ohh, right. So you can skip these by creating a `docs/nitpick-exceptions` file, and list this there as 
`py:class astropy.nddata.mixins.ndio.NDIOMixin`. Reason is some non straightforward inheritances, apparently the issue didn't solve itself over the years.

Example is in e.g ccdproc, https://github.com/astropy/ccdproc/pull/221

Reply:
(Also, doctests are run during the normal pytest test runs but not during the docs building job. The latter though is picky about links, and bases classes are usually picked up by their full path rather than by the simplest namespace where their docs page is generated. Ideally this is something the astropy intersphinx mapping should deal with, but apparently it doesn't)